src/diffusers/schedulers/scheduling_ddim_flax.py

```diff
@@ -216,6 +216,9 @@ def step(
         # - pred_sample_direction -> "direction pointing to x_t"
         # - pred_prev_sample -> "x_t-1"
 
+        # TODO(Patrick) - eta is always 0.0 for now, allow to be set in step function
+        eta = 0.0
```

Comment:
Rather than a step argument, maybe it would make more sense as a configuration property? Any additional arguments to `step` lead to this:

https://github.com/huggingface/diffusers/blob/ce31f83d8c44c977dcf8413c99b8a6cb7f189c07/src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py#L242-L249

Reply:
This one I think we can solve by just letting the user forward `**step_kwargs` . It should IMO not be in the config because `eta` is a parameter that one might want to change between forward passes (contrary to `offset_steps`)